main.go

```diff
@@ -334,30 +333,31 @@ func mainCore() error {
 
 	blockDataSavers = append(blockDataSavers, explore)
 
-	// Sync up with the blockchain
+	// Coordinated sync up both sqlite and postgres DBs with the network. This
+	// closure captures the RPC client and the quit channel.
 	getSyncd := func(updateAddys, updateVotes, newPGInds bool,
 		fetchHeight int64) (int64, int64, error) {
-		// Simultaneously synchronize the ChainDB (PostgreSQL) and the block/stake
-		// info DB (sqlite). Results are returned over channels:
+		// Simultaneously synchronize the ChainDB (PostgreSQL) and the
+		// block/stake info DB (sqlite). Results are returned over channels:
 		sqliteSyncRes := make(chan dbtypes.SyncResult)
 		pgSyncRes := make(chan dbtypes.SyncResult)
 
 		// Synchronization between DBs via rpcutils.BlockGate
 		smartClient := rpcutils.NewBlockGate(dcrdClient, 10)
 
-		// stakedb (in baseDB) connects blocks *after* ChainDB retrieves them, but
-		// it has to get a notification channel first to receive them. The BlockGate
-		// will provide this for blocks after fetchHeight.
+		// stakedb (in baseDB) connects blocks *after* ChainDB retrieves them,
+		// but it has to get a notification channel first to receive them. The
+		// BlockGate will provide this for blocks after fetchHeight.
 		baseDB.SyncDBAsync(sqliteSyncRes, quit, smartClient, fetchHeight)
 
-		// Now that stakedb is either catching up or waiting for a block, start the
-		// auxDB sync, which is the master block getter, retrieving and making
-		// available blocks to the baseDB. In return, baseDB maintains a
+		// Now that stakedb is either catching up or waiting for a block, start
+		// the auxDB sync, which is the master block getter, retrieving and
+		// making available blocks to the baseDB. In return, baseDB maintains a
 		// StakeDatabase at the best block's height.
 		go auxDB.SyncChainDBAsync(pgSyncRes, smartClient, quit,
 			updateAddys, updateVotes, newPGInds)
 
-		// Wait for the results
+		// Wait for the results from both DBs.
```

Comment:
They are actually three database especially when full mode is running.

Reply:
Referring to the agenda db?  `getSyncd` only runs the sync routines of `wiredDB` and `ChainDB`.  Note that this keeps the `StakeDatabase` at the same height as these too, but that is managed by `wiredDB` so I'm only mentioning the two DBs with sync routines.
Do we need to be doing something with agenda db on startup, while catching up?

Reply:
I would suggest we ignore agenda db for now as we have a lot of loose end that will be addressed when the new version of `getvoteinfo` rpc endpoint on dcrd is finalized.

Reply:
But updates are always checked on startup https://github.com/decred/dcrdata/blob/master/main.go#L288-L294